controllers/drplacementcontrol_controller.go

```diff
@@ -1568,9 +1574,11 @@ func (r *DRPlacementControlReconciler) updateResourceCondition(
 		drpc.Status.LastGroupSyncDuration = nil
 		drpc.Status.LastGroupSyncBytes = nil
 
-		r.setLastSyncTimeMetric(&drpcMetrics.SyncMetrics, nil, log)
-		r.setLastSyncDurationMetric(&drpcMetrics.SyncDurationMetrics, nil, log)
-		r.setLastSyncBytesMetric(&drpcMetrics.SyncDataBytesMetrics, nil, log)
+		if drpcMetrics != nil {
```

Comment:
This quick fix works, but for future fixes, we should resort back to the earlier PR (#1041). If you have a valid `DRPolicy` and `DRPC`, then you should always have valid `DRPCMetrics` and no need for the `if drpcMetrics != nil` everywhere it is used.